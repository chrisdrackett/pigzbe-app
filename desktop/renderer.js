import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
