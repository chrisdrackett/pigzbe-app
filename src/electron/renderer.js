import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
