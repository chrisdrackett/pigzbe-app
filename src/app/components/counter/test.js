import React from 'react';
import renderer from 'react-test-renderer';
import Counter from './';

describe('Counter', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Counter/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
