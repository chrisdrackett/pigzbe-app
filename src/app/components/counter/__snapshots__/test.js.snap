// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgb(255, 255, 255)",
      "borderRadius": 14.5,
      "flexDirection": "row",
      "height": 29,
      "justifyContent": "center",
      "opacity": 1,
      "paddingBottom": 3,
      "paddingLeft": 3,
      "paddingRight": 6,
      "paddingTop": 3,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/app/components/counter/images/wollo.png",
      }
    }
    style={
      Object {
        "height": 23,
        "width": 23,
      }
    }
  />
  <Image
    source={
      Object {
        "testUri": "../../../src/app/components/counter/images/times.png",
      }
    }
    style={
      Object {
        "height": 12,
        "marginLeft": 5,
        "marginRight": 5,
        "width": 12,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "rgb(0, 50, 120)",
        "fontFamily": "Poppins",
        "fontSize": 20,
        "fontWeight": "bold",
        "lineHeight": 26,
      }
    }
  />
</View>
`;
